loony/node.nim

```diff
@@ -69,6 +69,17 @@ else:
   template incEnqPathCounter*(): untyped = discard
   template incDeqPathCounter*(): untyped = discard
 
+template prn*(idx: uint16): uint16 =
+  ## prn = 'Pro re nata' - when required
+  ## Provides the actual index depending on
+  ## if we are rotating the index or not.
+  when loonyRotate:
```

Comment:
```suggestion
  when defined(loonyRotate):
```

Shouldn't this be the case?

Reply:
Since it's defined as a constant, it doesn't behave correctly when this change is implemented. You can see this is the case by compiling with a slotcount of 1023. This *should* fail, and it compiles successfully when the condition is defined(loonyRotate).